DOCKER/docker-compose

```diff
@@ -0,0 +1,34 @@
+version: "3.8"
+
+services:
+
+  postgis:
+    build:
+      context: .
+      dockerfile: dockerfile-toms-test-postgis
+    hostname: toms_postgis2
+    environment:
+      - POSTGRES_PASSWORD=postgis
+      - POSTGRES_DBNAME=gis
+    volumes:
+      - "pg_data_toms_test:/var/lib/postgresql"
+      - /home/QGIS/plugins/TOMs/DATAMODEL:/docker-entrypoint-initdb.d
+      - /home/QGIS/plugins/TOMs/test/data:/docker-entrypoint-initdb.d
+    ports:
+      - 5435:5432
```

Comment:
is it a typo (5435 instead of 5432) or is it on purpose ? (you were mentioning that postgis isn't exposed)

Reply:
I had understood that this is the port mapping from the container to the host. I am currently using 5432 for another db/container and so this seemed to be the way to add another container and access it. Does this seem sensible?